addons/skin.estuary/xml/View_51_Poster.xml

```diff
@@ -144,7 +137,7 @@
 				<texture fallback="DefaultMovies.png" background="true">$VAR[PosterThumbVar]</texture>
 			</control>
 			<control type="group">
-				<visible>String.IsEqual(ListItem.DBtype,tvshow) | String.IsEqual(ListItem.DBtype,set) | String.IsEqual(ListItem.DBType,season)</visible>
+				<visible>String.IsEqual(ListItem.DBtype,tvshow) | String.IsEqual(ListItem.DBtype,set) | String.IsEqual(ListItem.DBType,season) | ListItem.IsRecording | ListItem.HasReminder | ListItem.HasTimer | String.IsEqual(ListItem.IsCollection) | String.IsEqual(ListItem.HasVideoVersions) | ListItem.IsPlaying | String.IsEqual(ListItem.IsResumable) | ListItem.HasArchive | Integer.IsGreater(ListItem.Playcount,0)</visible>
```

Comment:
I think this is wrong here?
`String.IsEqual(ListItem.IsCollection) | String.IsEqual(ListItem.HasVideoVersions) | ListItem.IsPlaying | String.IsEqual(ListItem.IsResumable)`

Not tested but I think should be:
`ListItem.IsCollection | ListItem.HasVideoVersions | ListItem.IsPlaying | ListItem.IsResumable`

Reply:
That's right the `ListItem.IsCollection` could also be dropped as there is already `String.IsEqual(ListItem.DBtype,set)`

Reply:
Fixed.